src/Agent.Sdk/Agent.Sdk.csproj

```diff
@@ -3,15 +3,14 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
Can we suppress (or better -- fix) individual package ref mismatch warnings rather than disabling all of them?

Reply:
Yea, this will become irrelevant once we completely move to .net6. In the meantime, I can address the specific package warning in the next PR.